documentation/sphinx/source/installation.rst

```diff
@@ -101,6 +103,60 @@ In the context of SCT, it can be used:
 See https://github.com/neuropoly/sct_docker for more information.
 
 
+Using pip (experimental)
+************************
+
+SCT can be installed using pip, with some caveats:
+
+- The installation is done in-place, so the folder containing SCT must
+  be kept around
+
+- In order to ensure coexistence with other packages, the dependency
+  specifications are loosened, and it is possible that your package
+  combination has not been tested with SCT.
+
+  So in case of problem, try again with the reference installation,
+  and report a bug indicating the dependency versions retrieved using
+  `sct_check_dependencies`.
+
+
+Procedure:
+
+#. Retrieve the SCT code to a safe place
+
+   Clone the repository and hop inside:
+
+   .. code:: sh
+
+      git clone https://github.com/neuropoly/spinalcordtoolbox
+
+      cd spinalcordtoolbox
+
+#. Checkout the revision of interest, if different from `master`:
+
+   .. code:: sh
+
+      git checkout ${revision_of_interest}
+
+#. If numpy is not already on the system, install it, either using
+   your distribution package manager or pip.
+
+#. Install sct using pip
```

Comment:
do we want to specify Python version? (2.x for now)

Reply:
Maybe not:
- Python 2 is obsoleted at the end of this year. I think we should be a lot more serious about supporting Python 3.
- Most people will still install using the installer
- People who install using pip, and have a good reason to do so, will probably have valuable feedback from their use. If we state that the way to install is experimental and request feedback, it could help prioritize some areas of development?